src/components/layout/Header.vue

```diff
@@ -2,11 +2,30 @@
 <template>
   <header class="banner">
     <h1>Attendance Tracker</h1>
+    <div v-if="isAuthenticated">  <!--Only shows after login-->
+      
+      <div v-if="isProfessor">
+         <h2>{{this.getUser.name}}	&#9413; </h2>
+      </div>
+      
+      <div v-else>
+        <h2>{{this.getUser.name}}	&#9416; </h2>
+      </div>
+
+    </div>
   </header>
 </template>
 
 <script>
+import { mapGetters } from "vuex";
+
 export default {
-  name: "Header"
+  name: "Header",
+
+  computed: {
+    // Import the getters from the global store
+    ...mapGetters(["getUser", "isAuthenticated","isProfessor"]),
+  }
 }
+
 </script>
```

Comment:
Here's a css suggestion that makes it look more like a small icon in the top-right corner. I feel it makes it more clear that it is your user name since it looks like what most websites do (having a profile picture & your name in the top right corner). This is what it looks like:
![image](https://user-images.githubusercontent.com/43451024/99998098-85e94880-2d94-11eb-82df-02b406607e92.png)

```suggestion
</script>
<style lang="scss" scoped>
.profileButton {
  position:absolute;
  top:5px;
  right:5px;
  border:1px solid white;
  padding: 3px;
  font-size:1.1em;
}
</style>
```

Note that to use the suggestion you also have to add `class="profileButton"` to the div which has the `v-if="isAuthenticated"` and also add `style="position:relative"` on the `<header>` tag.